README.md

```diff
@@ -260,9 +260,13 @@ onEditChange(initialData, handleEditChange);
 
 ### Level 4: Enable Managing Blocks directly on your frontend ([TODO](https://github.com/collective/volto-hydra/issues/4))
 
-If you completed level 2 & 3 (made blocks clickable and enabled live updates) then the editor will automatically gain the management of blocks on the frontend using the quanta toolbar
-- Add blocks ([TODO](https://github.com/collective/volto-hydra/issues/27))
-- remove blocks ([TODO](https://github.com/collective/volto-hydra/issues/26))
+If you completed level 2 & 3 (made blocks clickable and enabled live updates) then the editor will automatically gain the management of blocks on the frontend using the quanta toolbar.
+
+With Quanta toobar, you can use following features:
```

Comment:
if possible it would be good to add a shadow to the bar so it stands out more. See what you can take from the quanta design to make it show up better against white.

Reply:
@djay  Will something like this works?

![Screenshot from 2024-06-28 08-37-06](https://github.com/collective/volto-hydra/assets/99530996/03675bc2-9b2d-4a31-b8b5-54d3f447b49f)